crrExistingObjects.js

```diff
@@ -1,9 +1,17 @@
 const { doWhilst, eachSeries, eachLimit, waterfall, series } = require('async');
-const { Logger } = require('werelogs');
+const werelogs = require('werelogs');
 const { ObjectMD } = require('arsenal').models;
 const metadataUtil = require('./CrrExistingObjects/metadataUtils');
 
-const log = new Logger('s3utils::crrExistingObjects');
+const logLevel = Number.parseInt(process.env.DEBUG, 10) === 1
```

Comment:
these changes seem out of place in the PR and might be better split into a new one.

Reply:
This is just an extension of the PR, and the change is required to have more information on the process. It was discussed before I included them.

Reply:
Agreed, it's a small complement to ease debugging that we can keep here IMO.

Reply:
kk no problem, I like to include this additions in the description so reviewers know what to expect.